src/Common/src/Interop/Windows/kernel32/Interop.GetVolumeInformation.cs

```diff
@@ -11,5 +11,10 @@ internal partial class Kernel32
     {
         [DllImport(Libraries.Kernel32, EntryPoint = "GetVolumeInformationW", CharSet = CharSet.Unicode, SetLastError = true, BestFitMapping = false)]
         internal static extern bool GetVolumeInformation(string drive, [Out]StringBuilder volumeName, int volumeNameBufLen, out int volSerialNumber, out int maxFileNameLen, out int fileSystemFlags, [Out]StringBuilder fileSystemName, int fileSystemNameBufLen);
+
+        [DllImport(Libraries.Kernel32, EntryPoint = "GetVolumeInformationW", CharSet = CharSet.Unicode, SetLastError = true, BestFitMapping = false)]
+        internal static unsafe extern bool GetVolumeInformation(string drive, char* volumeName, int volumeNameBufLen, int* volSerialNumber, int* maxFileNameLen, out int fileSystemFlags, char* fileSystemName, int fileSystemNameBufLen);
```

Comment:
I'd recommend creating a tiny wrapper which only takes a name (make this method private) so that we can reduce unsafe code to minimum

Reply:
On next PR i'll replace also for DriveInfo https://github.com/dotnet/corefx/pull/31763#discussion_r210064925, something like https://gist.github.com/MarcoRossignoli/cb53c616211e3d3785b4d2f06547ee9f#file-driveinfo-windows-cs-L28:L43. Do i create 2 helper(to avoid unuseful allocation of in/out parameters)?

Reply:
or simply passing null/defaul parameters on wrapper...

Reply:
It looks like you're not using this overload. Should it be removed?